e2e/test/scenarios/collections/uploads.cy.spec.js

```diff
@@ -281,16 +281,12 @@ function uploadFile(testFile, valid = true) {
       cy.findByText(testFile.humanName);
     });
 
-    cy.findByRole("status").within(() => {
-      cy.findByText("Start exploring").click();
-    });
+    cy.findAllByRole("status").findByText("Start exploring").click();
 
     cy.url().should("include", `/model/`);
     cy.findByTestId("TableInteractive-root");
   } else {
-    cy.findByRole("status").within(() => {
-      cy.findByText("Error uploading your File");
-    });
+    cy.findAllByRole("status").findByText("Error uploading your File");
```

Comment:
There is absolutely no reason to use `.within()` all over the place. The only valid use case for it is when you need to search for or assert on multiple different elements within one parent. Otherwise, `.findByText` is scoped to the previously yielded element(s).

cc @iethree